Photino.Native/Photino.Windows.DpiHelp.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Photino.Windows.DpiHelp.h"
+
+// =============================================
+//
+// Types
+//
+// ===================
+typedef UINT(WINAPI *_GetDpiForWindowFunc)(HWND hwnd);
+typedef DPI_AWARENESS_CONTEXT(WINAPI *_SetThreadDpiAwarenessContextFunc)(DPI_AWARENESS_CONTEXT dpiContext);
+typedef int(WINAPI *_GetSystemMetricsForDpiFunc)(int nIndex, UINT dpi);
+
+// =============================================
+//
+// Statics
+//
+// ===================
+static HMODULE s_user32 = nullptr;
+static bool s_initialized = false;
```

Comment:
This is a minor point, but you can use `s_user32 != nullptr` in place of `s_initialized` if you only set the variable at the end of the `InitDpiHelper` function; that is, use `HMODULE user32 = LoadLibrary(...)`.